libraries/libxml/source/org/pentaho/reporting/libraries/xmlns/writer/XmlWriterSupport.java

```diff
@@ -923,63 +757,15 @@ public void writeTextNormalized( final Writer writer,
    * @param transformNewLine true, if a newline in the string should be converted into a character entity.
    * @return the normalised string.
    */
-  public static String normalize( final String s,
-                                  final boolean transformNewLine ) {
+  public String normalize( final String s, final boolean transformNewLine ) throws IOException {
```

Comment:
Changing a method from static to non static is a breaking API change. It will break code that is compiled against pure 7.0 artifacts. That is a big NO NO in a minor release. 

Duplicating code is normally not a nice thing to do, but here you will have to do it.